include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_v2r4r2.hpp

```diff
@@ -44,6 +44,40 @@ __global__ void
 #endif // end of if (defined(__gfx908__) || defined(__gfx90a__))
 }
 
+template <typename GridwiseGemm,
+          bool HasMainKBlockLoop,
+          InMemoryDataOperationEnum CGlobalMemoryDataOperation,
+          typename Block2CTileMap,
+          typename AElementwiseOperation,
+          typename BElementwiseOperation,
+          typename CElementwiseOperation>
+__global__ void
+#if CK_USE_LAUNCH_BOUNDS
+    __launch_bounds__(CK_MAX_THREAD_PER_BLOCK, CK_MIN_BLOCK_PER_CU)
+#endif
+        kernel_gemm_xdlops_v2r4r2_simplified(typename GridwiseGemm::Argument karg,
```

Comment:
Do we still need the version without ABC elementwise ops above [here](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/956/files#diff-325efd84547132f6a10cb99736c02a47cc837d12b51de9cb33ab04f1c4211030R30)?